docs/src/main/asciidoc/deploying-to-kubernetes.adoc

```diff
@@ -441,6 +441,42 @@ quarkus.kubernetes.readiness-probe.initial-delay=20s
 quarkus.kubernetes.readiness-probe.period=45s
 ----
 
+=== Add hostAliases
+To add entries to a Pod's `/etc/hosts` file, just apply the following configuration:
```

Comment:
Is there a link to the Kubernetes documentation where one could see more info?

Reply:
Done.